backend/controllers/tweet.js

```diff
@@ -115,21 +115,33 @@ module.exports.getTweetsByHandle = async (req, res) => {
   try {
     // Fetch tweets by handle, sorted in desc order of createdAt
     const handle = req.params.handle;
-    const tweets = await Tweet.findAll({
-      include: [
-        {
-          model: User,
-          attributes: ["name", "handle"],
-          where: { handle: handle },
-          required: true,
-        },
-      ],
-      order: [["createdAt", "DESC"]],
-    });
-
-    return res.status(200).json({
-      data: { tweets: tweets },
-    });
+    const validHandle = await User.findOne({ where: { handle: handle } });
```

Comment:
Again, the database should handle this for us.
Additional query shouldn't be required.